packages/types/src/api/web/app/user.ts

```diff
@@ -7,3 +7,5 @@ export interface SetFlagRequest {
   flag: string
   value: any
 }
+
+export type AppSelfResponse = ContextUserMetadata | {}
```

Comment:
Why this empty body?

Reply:
This is a possible response - I didn't want to change any functionality, just document what it actually does!